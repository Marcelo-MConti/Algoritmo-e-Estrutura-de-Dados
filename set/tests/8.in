0
100 0
121 104 125 274 212 226 218 182 256 198 273 178 109 127 204 254 114 148 210 208 172 223 233 161 281 253 162 262 123 241 247 112 189 215 260 164 175 225 176 132 130 252 181 243 129 166 240 298 295 220 103 291 203 113 288 136 106 124 244 152 289 108 169 133 116 276 195 263 219 147 251 100 149 239 122 282 137 290 192 187 275 267 138 292 190 141 237 199 142 202 111 213 300 154 185 206 205 101 279 280

4
113
