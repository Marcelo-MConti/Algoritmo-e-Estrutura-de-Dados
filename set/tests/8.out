100, 101, 103, 104, 106, 108, 109, 111, 112, 114, 116, 121, 122, 123, 124, 125, 127, 129, 130, 132, 133, 136, 137, 138, 141, 142, 147, 148, 149, 152, 154, 161, 162, 164, 166, 169, 172, 175, 176, 178, 181, 182, 185, 187, 189, 190, 192, 195, 198, 199, 202, 203, 204, 205, 206, 208, 210, 212, 213, 215, 218, 219, 220, 223, 225, 226, 233, 237, 239, 240, 241, 243, 244, 247, 251, 252, 253, 254, 256, 260, 262, 263, 267, 273, 274, 275, 276, 279, 280, 281, 282, 288, 289, 290, 291, 292, 295, 298, 300, 
